common/journal_entries/00_sick_man.txt

```diff
@@ -0,0 +1,1007 @@
+			country_or_subject_owns_entire_state_region = STATE_WESTERN_THRACE
+			country_or_subject_owns_entire_state_region = STATE_EASTERN_THRACE
+			any_scope_state = {
+				state_region = s:STATE_MONTENEGRO
+			}
+			any_scope_state = {
+				state_region = s:STATE_DOBRUDJA
+			}
+			any_subject_or_below = {
+				any_scope_state = {
+					state_region = s:STATE_MOLDAVIA
+				}
+			}
+		}
+	}
+	on_complete = {
+		custom_tooltip = je_sick_man_advance
+		change_variable = {
+			name = sick_man_var
+			add = 1
+		}
+		add_radicals = {
+			culture = cu:turkish
+			value = 0.15
+		}
+		add_loyalists = {
+			culture = cu:greek
+			value = 0.15
+		}
+		add_loyalists = {
+			culture = cu:bulgarian
+			value = 0.15
+		}
+		add_radicals = {
+			culture = cu:albanian
+			value = 0.15
+		}
+		add_loyalists = {
+			culture = cu:serb
+			value = 0.15
+		}
+		add_radicals = {
+			culture = cu:mashriqi
+			value = 0.15
+		}
+		add_loyalists = {
+			culture = cu:armenian
+			value = 0.15
+		}
+		add_radicals = {
+			culture = cu:kurdish
+			value = 0.15
+		}
+		add_radicals = {
+			culture = cu:bosniak
+			value = 0.15
+		}
+		add_radicals = {
+			culture = cu:bedouin
+			value = 0.15
+		}
+		add_loyalists = {
+			culture = cu:croat
+			value = 0.15
+		}
+		add_loyalists = {
+			culture = cu:sephardic
+			value = 0.15
+		}
+		add_loyalists = {
+			culture = cu:georgian
+			value = 0.15
+		}
+		add_modifier = {
+			name = sick_man_liberal_secularism
+			years = 10
+		}
+		remove_modifier = internal_strife
+	}
+	invalid = {
+		custom_tooltip = {
+			text = finished_sick_man_tt
+			has_variable = sick_man_complete
+		}
+	}
+	transferable = no
+	weight = 5000
+}
+
+# Labor
+je_sick_man_liberal_labor = {
+	icon = "gfx/interface/icons/event_icons/event_industry.dds"
+	complete = {
+		scope:journal_entry = {
+			is_goal_complete = yes
+		}
+		NOR = {
+			has_law = law_type:law_serfdom
+			has_law = law_type:law_tenant_farmers
+		}
+		OR = {
+			has_law = law_type:law_interventionism
+			has_law = law_type:law_laissez_faire
+		}
+		has_law = law_type:law_slavery_banned
+	}
+	on_complete = {
+		set_variable = completed_sick_man_bureaucracy_je
+		custom_tooltip = je_sick_man_advance
+		change_variable = {
+			name = sick_man_var
+			add = 1
+		}
+		add_modifier = {
+			name = sick_man_liberal_labor
+			years = 10
+		}
+	}
+	invalid = {
+		custom_tooltip = {
+			text = finished_sick_man_tt
+			has_variable = sick_man_complete
+		}
+	}
+	transferable = no
+	weight = 5000
+}
+
+# Syria & Egypt
+je_sick_man_syria = {
+	icon = "gfx/interface/icons/event_icons/event_military.dds"
+	complete = {
+		owns_entire_state_region = STATE_ALEPPO
+		owns_entire_state_region = STATE_SYRIA
+		owns_entire_state_region = STATE_LEBANON
+		owns_entire_state_region = STATE_PALESTINE
+		owns_entire_state_region = STATE_TRANSJORDAN
+		owns_entire_state_region = STATE_ADANA
+		OR = {
+			has_law = law_type:law_agrarianism
+			has_law = law_type:law_interventionism
+		}
```

Comment:
Why are there law requirements here?

Reply:
It's about reforming the Vilayets, which require the Syrian Vilayet to be returned to the Empire. I can work to make that more clear to the player.